src/content/defi/index.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Decentralized finance (defi)
+description: An overview of defi on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+image: ../../assets/use-cases/defi.png
+sidebarDepth: 2
+summaryPoints:
+  [
+    "A global, open alternative to the current financial system.",
+    "Probably the biggest use case for Ethereum right now.",
+    "Powered by smart contracts on the Ethereum blockchain.",
+  ]
+---
+
+_Defi is an alternative to a financial system that many of us have grown to mistrust thanks to the cycles of mismanagement, crashes, and bail outs. It's an open global system that gives you more control and visibility over your money. It's exposure to global markets when your currency at home is failing. And there are no banker bonuses – it's largely owned by its users or maintained as a public good. And right now there's over 40 billion dollars worth of crypto in defi applications._
```

Comment:
> a financial system that many of us have grown to mistrust thanks to the cycles of mismanagement, crashes, and bail outs

I like this because it identifies a problem w/ the current system but I'm not sure how I feel about leading with those points. Are these the biggest problems in finance? In my mind, even if there wasn't mismanagement, crashes & bailouts, defi still offers massive improvements over the existing infrastructure, largely because it's infrastructure that's 100s of years old that still relies on archaic systems & processes. Defi is how the financial system would be designed if you built it from first principles in the internet age.

Reply:
> over 40 billion dollars worth of crypto in defi applications

"tens of billions of dollars"? Thinking on how we can avoid frequent updates to this 😅

Reply:
yeah, future proofing those numbers a little may be worth it 😂

And I agree w/ the above. Don't think it's essential to make these points right up front. Hesitate to put words in peoples mouths too with the "many of us have grown to mistrust."